app/logic/courseManagement.py

```diff
@@ -0,0 +1,34 @@
+from app.models.courseInstructor import CourseInstructor
+from app.models.courseStatus import CourseStatus
+from app.models.course import Course
+from app.models.term import Term
+
+
+def pendingCourses(termId):
+    '''
+    Queries the database to get all the neccessary information for
+    pending courses.
+    '''
+
+    pendingCourses = (CourseInstructor.select(Course.courseName, Term.description, CourseInstructor.user, Course.status)
+                    .join(Course)
+                    .join(CourseStatus)
+                    .switch(Course)
+                    .join(Term)).where(CourseInstructor.course.term.id == termId, Course.status.status != "Approve")
+
+
+    return pendingCourses
+def approveCourses(termId):
+    '''
+    Queries the database to get all the neccessary information for
+    pending courses.
+    '''
+
+    approveCourses = (CourseInstructor.select(Course.courseName, Term.description, CourseInstructor.user, Course.status)
+                    .join(Course)
+                    .join(CourseStatus)
+                    .switch(Course)
+                    .join(Term)).where(CourseInstructor.course.term.id == termId, Course.status.status == "Approve")

```

Comment:
Shouldn't this status be "Approved" instead of "Approve"? Or does it mean something other than that this course is approved?

Reply:
And if it should be approved, that language should change everywhere (including variable and method names)